Loenn/entities/lightningstrikes.lua

```diff
@@ -0,0 +1,63 @@
+local contorthelper = require("mods").requireFromPlugin("libraries.contorthelper")
+local xnaColors = require("mods").requireFromPlugin("libraries.xnacolorsforoptions")

```

Comment:
You're trying to load `libraries/xnacolorsforoptions.lua`, but that file definitely doesn't exist.